### PR TITLE
Make Kanopy Work... For Now...

### PR DESCRIPTION
**Pull request by lisahamlett**
### What are you adding/fixing?
Added a CONNECTION for https://emory.kanopy.com:443/ksa/tr?ksa_page_url=https%3A%2F%2Femory-kanopy-com.proxy.library.emory.edu%2F&ksa_page_title=Kanopy+-+Stream+Classic+Cinema%2C+Indie+Film+and+Top+Documentaries&ksa_http_referrer= because that is literally the only link showing in the logs and, hopefully, it will allow us to track who is using the resource.

#### Awesome List for Reviewers of Parsers :star::star:
- [ ] fetch new branch to local repo 'git pull'
- [ ] checkout new branch to test 'git checkout --track origin/{nameofbranch}' / make sure you're on correct branch with 'git status'
- [ ] find proxied resource and look through it (this generates entries in the logs).  Try to go through everything in nav bar since this will likely have most of the domain changes.
- [ ] go to the logs on the main proxy server and grep for your username.  Use these lines to then test.
- [ ] check output from parser directory:  'echo 'line from log' | ./parser.js
- [ ] check manifest.json to make sure no information is missing
- [ ] review/approve/merge pull request based on findings

### CODE REVIEW

File: kanopy/parser.js [hunk 1]
```diff
@@ -22,7 +22,11 @@ module.exports = new Parser(function analyseEC(parsedUrl, ec) {
 
   let match;
 
-  if ((match = /^\/wayf\/product\/(.*)$/i.exec(path)) !== null) {
+  if (/^\/ksa\/.*$/i.test(path)) {
```
Comment (AGCooper) [inline kanopy/parser.js]: I would suggest testing for /ksa/tr.

Comment (AGCooper) [reply kanopy/parser.js]: Also is ksa_page_title= something worth parsing for?

**Review by AGCooper — CHANGES_REQUESTED** [review]

File: kanopy/parser.js (hunk 1 above)
Comment (lisahamlett) [reply kanopy/parser.js]: That's the title of the homepage, so I dunno but I'll add it.  :)

**Comment by lisahamlett** [comment]
I updated the parser to match on /ksa/tr.
I do not know how to parse the page title...

**Comment by AGCooper** [comment]
Check Slack.

**Comment by lisahamlett** [comment]
Alright.  I believe this one is ready to go.

**Pull request merged**